app/assets/stylesheets/application.scss

```diff
@@ -17,21 +17,28 @@
 @import "bootstrap";
 
 body {
-  background-color: #e5e8e0;
+  background-color: #e0e0e0;
 }
 
-.container-fluid { max-width: 80%;
-
+.container-fluid {
+  max-width: 80%;
 }
 
-img {width: 100%;
-
+img {
+  width: 100%;
 }
 
 .featured {
   background-color: #c3a17d;
-  h3{color: white;
+  h3 {color: white;
     text-transform: uppercase;
+    text-align: center;
+    font-size: 3em;
+    padding-bottom: 20px;
+    a {
```

Comment:
Rule declaration should be preceded by an empty line<br>Selector should have depth of applicability no greater than 2, but was 3